File: iut/TP7_TP8/palindrome.c
int palindrome(char mot[])
{
    int i;
    int l;

    i = 0;
    l = 0;
    while (mot[l])
        l++;
    while (mot[i])
    {
        if (mot[i] != mot[l-1])
        {
            return 1;
        }
        i++;
        l--;
    }
    return 0;
}

#include <stdio.h>
int main(void)
{
    char mot[] = "test";
    printf("%i", palindrome(mot));
    return 0;
}

File: iut/TP7_TP8/palindrome.py
def palindrome(mot):

    i = 0
    l = 0
    le = len(mot)
    while (l != le):
        l+=1
    while (i != le):
    
        if (mot[i] != mot[l-1]):
        
            return 1
        i+=1
        l-=1
    return 0


mot = "test"
print(f"{palindrome(mot)}")

